src/Sylius/Behat/Context/Setup/ProductOptionContext.php

```diff
@@ -98,17 +103,49 @@ public function thisProductOptionHasTheOptionValueWithCode(
     }
 
     /**
+     * @param string $name
+     * @param string|null $code
+     * @param string|null $position
+     *
+     * @return ProductOptionInterface
+     */
+    private function createProductOption($name, $code = null, $position = null)
+    {
+        /** @var ProductOptionInterface $productOption */
+        $productOption = $this->productOptionFactory->createNew();
+        $productOption->setName($name);
+        $productOption->setCode($code ? $code : $this->generateCodeFromName($name));
+        $productOption->setPosition($position);
+
+        $this->sharedStorage->set('product_option', $productOption);
+        $this->productOptionRepository->add($productOption);
+
+        return $productOption;
+    }
+
+    /**
      * @param string $value
      * @param string $code
      *
      * @return ProductOptionValueInterface
      */
     private function createProductOptionValue($value, $code)
     {
+        /** @var ProductOptionValueInterface $productOptionValue */
         $productOptionValue = $this->productOptionValueFactory->createNew();
         $productOptionValue->setValue($value);
         $productOptionValue->setCode($code);
 
         return $productOptionValue;
     }
+
+    /**
+     * @param string $name
+     *
+     * @return string
+     */
+    private function generateCodeFromName($name)
+    {
+        return StringInflector::nameToCode($name);
```

Comment:
`$this->generateCodeFromName($name)` is even longer than `StringInflector::nameToCode($name)`, why extracting this?